Xamarin.Forms.Core/Shell/Shell.cs

```diff
@@ -487,7 +484,7 @@ internal static void ApplyQueryAttributes(Element element, IDictionary<string, s
 				filteredQuery.Add(key, q.Value);
 			}
 
-			if (baseShellItem != null)
+			if (baseShellItem != null && baseShellItem is ShellContent)
```

Comment:
isn't this redundant ?
```suggestion
			if (baseShellItem is ShellContent)
```